lib/Partitioner/Partitioner.cpp

```diff
@@ -84,7 +91,8 @@ void Partitioner::dumpDAG(llvm::StringRef dotFilename) const {
     auto *root = nodes[i];
     for (size_t j = 0; j < root->children.size(); j++) {
       auto child = root->children[j];
-      myfile << root->name << " -> " << child->name << ";";
+      myfile << escapeDottyString(root->name) << " -> "
```

Comment:
You probably need to add `\""` before and after each of these `escapeDottyString` calls, so that it produces `"Escaped string"`.